runtime/oti/VMHelpers.hpp

```diff
@@ -1060,6 +1060,20 @@ class VM_VMHelpers
 		return rc;
 	}
 
+	/**
+	 * Determine if a RAM static field ref is resolved.
+	 *
+	 * @param flagsAndClass[in] field from the ref
+	 * @param valueOffset[in] field from the ref
+	 *
+	 * @returns true if resolved, false if not
+	 */
+	static VMINLINE bool
+	staticFieldRefResolved(IDATA flagsAndClass, UDATA valueOffset)
+	{
+		return ((UDATA)-1 != valueOffset) && (flagsAndClass > 0);
```

Comment:
This helper is a bit of an attractive nuisance as it doesn't differentiate between resolved for put vs get.
See the `putstatic` code in the BytecodeInterpreter: https://github.com/eclipse/openj9/blob/0a25dca436291a5bc981d891b32b4fa763ca487f/runtime/vm/BytecodeInterpreter.hpp#L6260-L6261

Reply:
There isn't a tag for get, so it's really put vs get/dontknoworcare. I can add a bool for put and use is in the interpreter as well if you like.

Reply:
+1 to adding the bool - will hopefully make users think about the purpose and if they need put vs don'tcare semantics

Reply:
In deference to having to think about it, I won't add a default value for the bool.